### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1084] [Galaxy Trivia] [answered]
🟦☀️🟥🟥
💙💛❤️🤍💚
🟧⬜🟠🟩

**Comment by peterbaccaro** [comment]
1. **Republic of China**
2. **Seychelles**
3. **Niger**

![image](https://github.com/ParanoidUser/yolo/assets/74666079/2a04723f-eb9f-4ea5-8c9d-15f385490956)

**Comment by thuongtruong1009** [comment, marked as answer]
It appears to represent the flag of Ukraine 🇺🇦. The blue and yellow represent the national colors of Ukraine.
